src/lib.rs

```diff
@@ -185,16 +187,17 @@ pub trait CpuIdReader: Clone {
 }
 
 impl<F> CpuIdReader for F
-where
-    F: Fn(u32, u32) -> CpuIdResult + Clone,
+    where
+        F: Fn(u32, u32) -> CpuIdResult + Clone,
```

Comment:
is this from cargo fmt?

Reply:
I will check, but I would bet on my IDE running rustfmt. (JetBrain’s Rust Rover)